L1Trigger/L1THGCalUtilities/python/clustering3d.py

```diff
@@ -52,11 +52,11 @@ def create_histoMax(process, inputs,
     producer = process.hgcalBackEndLayer2Producer.clone(
             InputCluster = cms.InputTag('{}:HGCalBackendLayer1Processor2DClustering'.format(inputs))
             )
-    producer.ProcessorParameters.C3d_parameters = histoMax_C3d_clustering_params.clone()
+    producer.ProcessorParameters.C3d_clustering_parameters = histoMax_C3d_clustering_params.clone()
```

Comment:
Still not correct I think: should be `producer.ProcessorParameters.C3d_parameters.histoMax_C3d_clustering_parameters`
(here and elsewhere)

Reply:
I see, thanks.